### PR TITLE
Module 7 Project Exercise

### PR DESCRIPTION
**Pull request by pat-morris**

### CODE REVIEW

File: .env.template [hunk 1]
```diff
@@ -5,6 +5,7 @@ FLASK_ENV=development
 # Change the following values for local development.
 TRELLO_API_KEY=
 TRELLO_API_TOKEN=
+TRELLO_API_BASE_URL=
```
Comment (hugh-emerson) [inline .env.template:8]: You've added this here and to the pipeline, but it's not actually used by your code anywhere, right?
You still do `self.trello_web_url = 'https://api.trello.com/1'`.

You should either update your code to actually use it, or remove all references to TRELLO_API_BASE_URL

File: .dockerignore [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
+.vscode
+env
 .env
 .env.template
 .env.test
 .gitignore
-.vagrant/
-env/
+applogs.txt
+poetry.lock
```
Comment (hugh-emerson) [inline .dockerignore]: You shouldn't dockerignore poetry.lock. It's used for controlling specific versions of dependencies so you don't get any mismatch between installations from the same version of the codebase.

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+script:
+  - docker run test tests
+  - docker run test tests_integration
+  - >
+    docker run
+    -e TRELLO_API_BASE_URL
+    -e TRELLO_API_KEY
+    -e TRELLO_API_TOKEN
+    -e TRELLO_BOARD_ID
```
Comment (hugh-emerson) [inline .travis.yml]: You don't actually need to pass this one in because the test creates a test board and sets this based on the test board's id.

**Review by hugh-emerson — APPROVED** [review]
Looks good! I've just left some minor comments. Merge whenever you're ready.

File: .dockerignore (hunk 2 above)
Comment (pat-morris) [reply .dockerignore]: Removed poetry.lock from .dockerignore

File: .travis.yml (hunk 3 above)
Comment (pat-morris) [reply .travis.yml]: Removed trello_board_id from .travis.yml

**Pull request merged**